CHANGELOG.rst

```diff
@@ -1,3 +1,25 @@
+**3.0.0 - 08/12/24
+
+Breaking changes:
+  - Update components to use new Vivarium lookup table configuration
+  - Refactor Risk and RiskEffect components
```

Comment:
The RiskEffect component wasn't changed, but there is a new component in the effect.py file